src/components/confirm/confirm.ts

```diff
@@ -12,15 +27,15 @@ function useNativeConfirm() {
         && window.confirm;
 }
 
-async function nativeConfirm(options) {
+async function nativeConfirm(options: string | ConfirmOptions) {
     if (typeof options === 'string') {
         options = {
             title: '',
             text: options
-        };
+        } as ConfirmOptions;
     }
 
-    const text = (options.text || '').replaceAll('<br/>', '\n');
+    const text = (options.text ?? '').replace('<br/>', '\n');
```

Comment:
No need to change `||` to `??` unless necessary. `??` produces more code when transpiled (_we need to support older browsers that don't support `??`_).

Reply:
Fixed